sim/src/pandemic/mod.rs

```diff
@@ -6,14 +6,14 @@ use std::ops;
 
 use anyhow::Result;
 
-pub use pandemic::{Cmd, PandemicModel};
+pub use model::{Cmd, PandemicModel};
 use rand::Rng;
 use rand_distr::{Distribution, Exp, Normal};
 use rand_xorshift::XorShiftRng;
 
 use geom::{Duration, Time};
 
-mod pandemic;
```

Comment:
Ah, it's like Go's "no stuttering type names" rule. Nice that clippy has it too.